### PR TITLE
Replaced scp2-2022 library with ssh2-sftp-client

### PR DESCRIPTION
**Pull request by v-gayatrij**
Task name: Ansible

Description:  

- Migrated from scp2-2022 library to ssh2-sftp-client
- Downgraded ssh2 to 1.4.0

Documentation changes required: (Y/N) N

Added unit tests: (Y/N) N

Attached related issue: (Y/N)

Checklist:

- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by v-mohithgc — APPROVED** [review]
LGTM

**Review by LeftTwixWand — APPROVED** [review]
Looks good👍

File: Extensions/Ansible/Src/Tasks/Ansible/task.json
Comment (v-nagarajku) [inline Extensions/Ansible/Src/Tasks/Ansible/task.json:1]: Why "minimumAgentVersion" attribute is removed?

Comment (v-gayatrij) [reply Extensions/Ansible/Src/Tasks/Ansible/task.json:1]: "minimumAgentVersion" attribute was not present in the current prod version (i.e. 0.0.6 which is working fine). This attribute was added later this year without any testing done by the PR creator. Hence, we have removed this attribute to match the prod version. The test went fine without this attribute, and I hope this won't cause any issues.

**Review by v-nagarajku — APPROVED** [review]
LGTM

**Comment by PavelPikat** [comment]
@v-gayatrij @LeftTwixWand @v-nagarajku this change broke the extension for us - we no longer able to copy files to remote VM. 
Are there any changes required to Ansible playbooks and relative paths to make it work?

**Comment by v-gayatrij** [comment]
@PavelPikat the issue will be resolved by version 0.228.1 which is out now. Please check and confirm back.

**Comment by v-mohithgc** [comment]
Hi @PavelPikat, can you please mail us the logs of failure run (ie 0.228.0)

**Pull request merged**